src/EncodeHQ-CBR/EncodeParams.cpp

```diff
@@ -126,8 +126,8 @@ ProgramParams getCommandLineParams(int argc, char * argv[], const char * details
     const int fragment_length = cla_fragmentLength.getValue();
 
     // Check for valid combinations of parameters and options
-    if ((chromaFormat==RGB) && (cla_lumaDepth.isSet() || cla_chromaDepth.isSet()))
-      throw invalid_argument("luma/chroma depth is not appropriate for RGB (use -z or --bitDepth)");
+    // if ((chromaFormat==RGB) && (cla_lumaDepth.isSet() || cla_chromaDepth.isSet()))
+    //   throw invalid_argument("luma/chroma depth is not appropriate for RGB (use -z or --bitDepth)");
```

Comment:
Either remove or do not. Don't leave code commented out in a PR for merging.

Reply:
All gone!